### PR TITLE
add kimai2 to one click apps

### PR DESCRIPTION
**Pull request by lcx**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: kimai2.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          defaultValue: $$cap_gen_random_hex(32)
+    instructions:
+        start: |-
+           With Kimai, the boring process of feeding Excel spreadsheets with your working hours is not only simplified, it also offers dozens of other exciting features that you don't even know you're missing so far!
+        end: |-
+            Aaaand you're done! 😄
+            Your service is available at http://$$cap_appname.$$cap_root_domain
+            -------------------------------------------------------------------
+    displayName: Kimai
+    isOfficial: true
```
Comment (githubsaturn) [inline kimai2.yml:93]: This is not the official image. It's from a 3rd party repo: https://github.com/tobybatch/kimai2
The original repo is here: https://github.com/tobybatch/kimai2

File: kimai2.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            MYSQL_USER: $$cap_mariadb-user
+            MYSQL_PASSWORD: $$cap_mariadb-pass
+            MYSQL_ROOT_PASSWORD: $$cap_maridb-rootpass
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        restart: unless-stopped
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: lcxat/nginx-fpm-reverse-proxy
```
Comment (githubsaturn) [inline kimai2.yml:34]: This is using an implicit `latest` tag. Let's fix the version to prevent it from breaking the installations with future updates.

**Review by githubsaturn — COMMENTED** [review]
Thanks for the PR! Just a few minor comments.

File: kimai2.yml (hunk 1 above)
Comment (lcx) [reply kimai2.yml:93]: oh yes, you are right. Kimai does refer to the third party link https://www.kimai.org/documentation/docker.html but never states it's the "official".

File: kimai2.yml (hunk 2 above)
Comment (lcx) [reply kimai2.yml:34]: yep, never considered to check the :latest on my own containers. Let me fix that as well.

**Comment by lcx** [comment]
@githubsaturn both issues fixed. ✅

**Pull request merged**